draft-wang-ppm-differential-privacy.md

```diff
@@ -322,28 +408,42 @@ with the correct noise type.
 
 > TODO: Chris P: we will mention Prio3SumVec because that's what we use to describe aggregator DP with amplification
 
+# Primitives for Implementing DP Mechanisms {#primitives}
 
-# Differential Privacy Mechanism {#dp mechanism}
+This section describes various primitives required for implementing DP
+mechansims. The algorithms are designed to securely expand a short, uniform
+random seed into a sample from a given distribution.
 
-This section should define a framework to apply different dp noise type to a
-client measurement and/or an aggregation.
+## Discrete Laplace
 
-> TODO: Junye to spell out the interface of dp mechanisms, "Noise Mechanisms to Accomplish DP"
+> TODO: Specify a Laplace sampler
+
+## Discrete Gaussian
+
+> TODO: Specify a Gaussian sampler
+
+## Randomized Response
 
+> TODO: Specify any primitives required for randomized response mechanisms
 
-> TODO: Chris P: can we talk about primitives like sampling gaussian noise here,
-or do we just mention the various types briefly and detail in a separate section?
+# DP Mechanisms for VDAFs {#mechanisms}
 
-### Discrete Laplace
+The section defines a generic interface for DP mechanisms VDAFs. Compatible
+mechnamsms include
```

Comment:
This sentence is not finished?

Reply:
Good catch, I forgot to finish this bit. I just deleted the second sentence and the paragraph below.